addon/templates/components/light-field.hbs

```diff
@@ -18,23 +31,9 @@
         leaveEditMode=(action 'leaveEditMode')
         update=(action 'update')
       )
-    )
-    control=(component control
-      id=controlId
-      value=(get model fieldName)
-      onfocus=(action 'enterEditMode')
-      onblur=(action 'leaveEditMode')
-      update=(action 'update')
-    )
-    label=label
-    controlId=controlId
-    value=(get model fieldName)
-    errors=errors
-    isDisplayingErrors=isDisplayingErrors
-    enterEditMode=(action 'enterEditMode')
-    leaveEditMode=(action 'leaveEditMode')
-    update=(action 'update')
-  )}}
+      template
+    }}
+  {{/component}}
 {{else}}
   {{component component
```

Comment:
@tempo22 Shouldn't it be `template` here?

Reply:
👍